charts/redpanda/README.md

```diff
@@ -2,42 +2,44 @@
 
 [![Artifact Hub](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/redpanda-data)](https://artifacthub.io/packages/search?repo=redpanda-data)
 
-This Helm chart (`redpanda`) deploys a Redpanda cluster.
-Once deployed, you continue to use the Helm command and override values to change and/or upgrade your Redpanda deployment.
-The defaults are in [values.yaml][values].
-
-## Requirements
-
-- Helm v3.5.0 or newer
-- Kubernetes 1.21.0 or newer
-- Cert-manager 1.9.0 or newer (optional for TLS support)
-
-## Overview
-
-This is the Helm Chart for [Redpanda](https://redpanda.com). It provides the ability to set up a multi node redpanda cluster with the following optional features:
+The Redpanda Helm chart deploys a Redpanda cluster in Kubernetes, and provides the following features:
 
 - Schema registry (enabled by default)
 - REST (aka PandaProxy, enabled by default)
 - TLS
 - SASL
 - External access
 
-See the [examples folder][examples] with more details on how to use this helm chart.
-Each example focuses on specific features like the ones listed above.
-We recommend completing the instructions in the [60-Second Guide for Kubernetes][kubernetes-qs-dev] before continuing steps in any of these examples.
+## Requirements
 
-The [values.yaml][values] file is documented throughout.
-Please see this file for more details.
+- Helm version 3.6.0 or later
+- Kubernetes version 1.21.0 or later
+- Cert-manager 1.9.0 or later (only if you want TLS support)
 
 ## Installation
 
-See the [60-Second Guide for Kubernetes][kubernetes-qs-dev]
+To get started, see the [Redpanda documentation][kubernetes-qs-dev].
+
+For examples of using the Helm chart, see the [`examples/` directory][examples]. Each example focuses on a specific feature.
+
+## Configuration
+
+The Redpanda Helm chart is configured in the [`values.yaml`][values] file. You can override the default values in this file to customize your deployment. For example, you can do the following:
```

Comment:
Though technically true, this leads novice users into thinking that they need to modify this file. What is the goal of this line? Maybe we can think of better phrasing.

Reply:
We link to the Helm docs for details about overriding the defaults.

Maybe we can be explicit and say `To customize your deployment, you can override the default values in your own YAML file or in the command line by using the `--set` option.

Reply:
Yes, that sounds much better.